spec/index.bs

```diff
@@ -740,8 +740,10 @@ its mapping.
 
 {{GPUBuffer|GPUBuffers}} can be created via the following functions:
 
- - {{GPUDevice/createBuffer(descriptor)|GPUDevice.createBuffer(descriptor)}} that returns a new unmapped buffer.
- - {{GPUDevice/createBufferMapped(descriptor)|GPUDevice.createBufferMapped(descriptor)}} that returns a mapped buffer and its mapping.
+ - {{GPUDevice/createBuffer(descriptor)|GPUDevice.createBuffer(descriptor)}}
+     that returns a new unmapped buffer.
```

Comment:
could be phrase it "returns a new buffer in the XXX state" that properly links to the state definitions?